README.md

```diff
@@ -30,13 +30,42 @@ This blueprint assumes that you are familiar with GKE and
 To deploy this blueprint you need:
 
 - A [Google Cloud project](https://cloud.google.com/docs/overview#projects) with billing enabled.
-- An account with the [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles) on the project.
 - The `serviceusage.googleapis.com` must be enabled on the project. For more
   information about enabling APIs, see
   [Enabling and disabling services](https://cloud.google.com/service-usage/docs/enable-disable)
 - A Git repository to store the environment configuration.
 
-You create the infastructure using Terraform. The blueprint uses a local [Terraform backend](https://www.terraform.io/docs/language/settings/backends/configuration.html),
+### Service Account Roles & Permissions
+
+You can choose between **Project Owner** access (full access) or **Granular Access** for more fine-tuned permissions.
+
+#### Option 1: [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles)
```

Comment:
Can you remove this link from the title, and put it on line 46?

Reply:
Sure